.github/workflows/build-test.yaml

```diff
@@ -1516,9 +1535,29 @@ jobs:
           ./bin/kots admin-console push-images ./hack/tests/small.airgap ttl.sh/automated-${{ github.run_id }}
 
 
+  validate-kots-admin-console-generate-manifests:
+    runs-on: ubuntu-20.04
+    needs: [ enable-tests, can-run-ci, build-kots, build-kurl-proxy, build-migrations, push-minio, push-mc, push-rqlite ]
+    strategy:
+      fail-fast: false
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: download kots binary
+        uses: actions/download-artifact@v2
+        with:
+          name: kots
+          path: bin/
+      - run: chmod +x bin/kots
+      - name: run kots admin-console generate-manifests without k8s context
+        run: |
+          set +e
+          ./bin/kots admin-console generate-manifests --shared-password password
```

Comment:
maybe even attempt to deploy them and validate that kotsadm runs?

Reply:
agreed.  might play around with this test a bit since right now it's cli-only (no k8s).  could probably introduce k8s and just unset `KUBECONFIG` temporarily to test the no-context permutation.